transport/src/main/java/io/netty/channel/embedded/EmbeddedChannel.java

```diff
@@ -111,13 +111,26 @@ public EmbeddedChannel(boolean hasDisconnect, ChannelHandler... handlers) {
     }
 
     /**
+     * Create a new instance with the pipeline initialized with the specified handlers.
+     *
+     * @param register {@code true} if this {@link Channel} is directly registered on the {@link EventLoop}.
```

Comment:
`directly registered on the {@link EventLoop}` -> `registered to the {@link EventLoop} in the constructor`